routes/ses.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+const express = require("express");
+const bodyParser = require("body-parser");
+
+const router = express.Router();
+const jsonParser = bodyParser.json();
+
+
+router.post("/notification", jsonParser, async (req, res) => {
+  try {
+    console.log("ses-notification request body: ", req.body);
+
+    res.status(200).json(
+      {status: "OK"}
+    );
+  } catch (e) {
+    res.status(500).json(
+      {info: "Internal error."}
+    );
```

Comment:
Do we have some server logging enabled somewhere that will tell us if we ever have cases of 500s? Curious how the `.catch()` would ever be reached even, since the method doesn't really _do_ anything or process the `req.body` or whatever.

Reply:
We have heroku & papertrail logging on heroku. I'm sure @Micheletto and Ops team have their own response logging too.

This is still WIP so no functional code in here yet. Just making sure we're getting notifications from SES.